File: platform/sdk/nodejs/examples/cloud/cloud-028_lifecycle_bucket_stats.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("stats-bucket");
const result = client.lifecycleBucketStats("stats-bucket");
if (!result.stats.totalObjects || result.stats.totalBytes <= 0) process.exit(1);
console.log("XJ-OK: cloud-028_lifecycle_bucket_stats");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-028_lifecycle_bucket_stats.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("stats-bucket")
result = client.lifecycleBucketStats("stats-bucket")
if not result["stats"]["totalObjects"] or result["stats"]["totalBytes"] <= 0:
    sys.exit(1)
print("XJ-OK: cloud-028_lifecycle_bucket_stats")
sys.exit(0)
